operators/dream_texture.py

```diff
@@ -105,7 +105,9 @@ def done_callback(future):
             result: ImageGenerationResult = future.result(last_only=True)
             for i, result_image in enumerate(result.images):
                 seed = result.seeds[i]
-                image = bpy_image(str(seed), result_image.shape[1], result_image.shape[0], result_image.ravel(), last_data_block)
+                prompt_string = context.scene.dream_textures_prompt.prompt_structure_token_subject
+
+                image = bpy_image(f"{prompt_string} ({seed})", result_image.shape[1], result_image.shape[0], result_image.ravel(), last_data_block)
```

Comment:
Blender truncates image names to 63 characters, which means if the prompt is long the seed will be dropped automatically. Two possible solutions:
* Swap so the prompt is in parentheses after the seed
* Check the length of the seed and trim the `prompt_string` accordingly so it all fits - If Blender increases this limit in the future though we'd have to remember to update on our end
```suggestion
                image = bpy_image(f"{seed} ({prompt_string})", result_image.shape[1], result_image.shape[0], result_image.ravel(), last_data_block)
```

Reply:
The only issue I have with putting the seed first is that we lose the default alphabetic sorting within blender and it becomes less organized when selecting images in the Image Editor.

One solution to that could be to use a counter, like how the Stable Diffusion WebUI does it, which would then chronologically order the outputs. Another benefit to this is that you will not overwrite duplicate named images.

Otherwise I think for now trimming the names length to 63 minus the seeds string length seems like a good solution.

Reply:
```
seed_str_length = len(str(seed))
trim_aware_name = (prompt_string[:58 - seed_str_length] + '..') if len(prompt_string) > 58 else prompt_string
name_with_trimmed_prompt = f"{trim_aware_name} ({seed})"
```
![image](https://user-images.githubusercontent.com/6715569/213838158-9a4bdb78-f19d-4e03-a630-eb62e943f888.png)

I've come up with this which seems to work quite nicely. Commit incoming.